appserver/tests/payara-samples/samples/concurrency/pom.xml

```diff
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>jakarta.platform</groupId>
             <artifactId>jakarta.jakartaee-api</artifactId>
```

Comment:
Should this not be using the web api now?

Reply:
this was removed